src/osvr/Server/ConfigFilePaths.cpp

```diff
@@ -0,0 +1,79 @@
+// Internal Includes
+#include <osvr/Util/PlatformConfig.h>
+#include <osvr/Server/ConfigFilePaths.h>
+#include <osvr/Server/ConfigureServerFromFile.h>
+#include <osvr/Util/GetEnvironmentVariable.h>
+
+// Library/third-party includes
+#include <boost/filesystem.hpp>
+
+// Standard includes
+#include <vector>
+
+namespace osvr {
+    namespace server {
+
+        inline std::string getUserConfigDirectory() {
+            namespace fs = boost::filesystem;
+            using osvr::util::getEnvironmentVariable;
+
+            fs::path configDir;
+            std::string configSubpath = "config";
+
+#if defined(OSVR_LINUX)
+            // There's currently no great location for storing log files in the
```

Comment:
We can delete the first paragraph of the comment and update the second para to reflect `XDG_CONFIG_HOME`.